src/parser/paladin/shared/spells/azeritetraits/GraceOfTheJusticar.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS/index';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import ItemHealingDone from 'interface/others/ItemHealingDone';
+import Events from 'parser/core/Events';
+
+import BeaconHealSource from '../../../holy/modules/beacons/BeaconHealingDone.js';
+
+
+/**
+ * Grace of the Justicar
+ * Judging a foe heals up to 10 allies within 8 yards of that enemy for 899. (@iLvl 400)
+ * Example Log: https://www.warcraftlogs.com/reports/kMbVanmJwCg7WrAz#fight=last&type=summary&source=2
+ */
+class GraceOfTheJusticar extends Analyzer {
+  static dependencies = {
+    beaconHealSource: BeaconHealSource,
```

Comment:
Seeing as this depends on a Holy module, unless you want to make this specific to the Holy implementation and want to support this trait for Ret/Prot, this should be in the `holy` spec folder instead of shared. You can make an `azeritetraits` folder in Holy if you want.

Reply:
I tested that and  trait still provides healing when you swap out of holy, so I put it in shared.  My thought being, the non-beacon specific events will still fire and let users know how much healing the trait is doing.

Reply:
BeaconHealSource does not exist on the other specs (and should not exist), so if you want to do that you'd have to make that logic specific to the Holy module. You can extend the module if that's what you want, same way as we do for several core modules.

Reply:
Okay, that makes sense, I'll change it to the holy folder.
The crazy relative path needed to import the beacon event was my first clue that things were not as they should be.